tests/testing/LocalBearTestHelperTest.py

```diff
@@ -26,6 +30,22 @@
                               valid_files=files,
                               invalid_files=())
 
+language = str(TestBear.languages[0])
+aspect_list = [get_aspect('Indentation')(language, allow_trailing_spaces=True)]
+
+invalidAspectTest = verify_local_bear(TestBear,
+                                      valid_files=(),
+                                      invalid_files=files,
+                                      settings={'result': False},
+                                      aspects=AspectList(aspect_list),
```

Comment:
@jayvdb prior to this design, I had tried to make the aspect_list in `verify_local_bear` only, doing something like this
```
aspect_list = []
for aspect_setting, value in aspects.items():
    aspect, taste = aspect_setting
    aspect_list.append([get_aspect(aspect)(language, taste=value)])
aspects = AspectList(aspect_list)
```
But aspects are not interpreting this to `AspectList([get_aspect('TrailingSpace')(language, allow_trailing_spaces=True)])` while passing `aspect_list={('TrailingSpace', 'allow_trailing_spaces'): True}` from the tests. This is because `taste=value` is interpreting as `taste=True` not `allow_trailing_spaces=True`, even tried string `'allow_trailing_spaces'=True` gives an error that you can't assign to a string literal.  Even tried passing dictionary to it but no good. And we can't set the tastes after creating the aspect object in the current aspect core.

I spent 4 hours brainstorming on this, but of no use. That is why I redesigned the whole thing. And this is the design we have been following for tests in mapping functions like `map_setting_to_aspect` as well (https://github.com/coala/coala/blob/2843072eeb0d63d31958e6680c074c6fcfe0ff8f/tests/bearlib/aspects/DecoratorsTest.py#L47).